api/v1/tag_test.go

```diff
@@ -35,7 +35,7 @@ func TestFindTagListFromMemoContent(t *testing.T) {
 		},
 		{
 			memoContent: "#tag1 http://123123.com?123123#tag2 \n#tag3  #tag4 http://123123.com?123123#tag2) ",
-			want:        []string{"tag1", "tag2", "tag2)", "tag3", "tag4"},
+			want:        []string{"tag1", "tag2", "tag3", "tag4"},
 		},
```

Comment:
It'll be better to add more test cases.

Reply:
Just added one more test and fixed the test logic not checking the values of the slices.

Reply:
@pipe01 I will share more info about multi-level tags with you: I've just pulled the latest code and built an image using the main branch, however multi-level tags are still not rendering properly.
Would you please check it again?  Thank you very much.
#2009 #2024